ci-operator/step-registry/gather/azure-cli/gather-azure-cli-commands.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export PATH=$PATH:/tmp/bin
+mkdir -p /tmp/bin
+
+echo "$(date -u --rfc-3339=seconds) - Installing tools..."
+
+# install jq
+# TODO move to image
+curl -sL https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 > /tmp/bin/jq
+chmod ug+x /tmp/bin/jq
+
+# install newest oc
+curl https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz | tar xvzf - -C /tmp/bin/ oc
+chmod ug+x /tmp/bin/oc
+
+
+# set the parameters we'll need as env vars
+AZURE_AUTH_LOCATION="${CLUSTER_PROFILE_DIR}/osServicePrincipal.json"
+AZURE_AUTH_CLIENT_ID="$(cat ${AZURE_AUTH_LOCATION} | jq -r .clientId)"
+AZURE_AUTH_CLIENT_SECRET="$(cat ${AZURE_AUTH_LOCATION} | jq -r .clientSecret)"
+AZURE_AUTH_TENANT_ID="$(cat ${AZURE_AUTH_LOCATION} | jq -r .tenantId)"
+
+CLUSTER_NAME="$(oc get -o jsonpath='{.status.infrastructureName}' infrastructure cluster)"
+echo "Cluster name: $CLUSTER_NAME"
+CLUSTER_VERSION="$(/tmp/bin/oc adm release info -o json | jq -r .metadata.version)"
```

Comment:
nit: maybe you don't need `/tmp/bin/`? since the other `oc` commands don't use it.

Reply:
Probably picking up a bad idea from the step I was working off when I did this. If I end up back in here to improve something I will address.